diff --git a/webpack.config.ts b/webpack.config.ts
index 20fccc1..003bf5b 100755
--- a/webpack.config.ts
+++ b/webpack.config.ts
@@ -76 +76 @@ function glob_script_files() {
-  return results;
+  return ['util/mvu_zod.ts'];
@@ -563,15 +563 @@ function parse_configuration(entry: Entry): (_env: any, argv: any) => webpack.Co
-      const cdn = {
-        sass: 'https://jspm.dev/sass',
-      };
-      const package_json = JSON.parse(fs.readFileSync(path.join(import.meta.dirname, 'package.json'), 'utf-8')) as {
-        dependencies?: Record<string, string>;
-        devDependencies?: Record<string, string>;
-      };
-      const package_versions = { ...package_json.devDependencies, ...package_json.dependencies };
-      const version = package_versions[request]?.replace(/^[~^]/, '');
-      const versioned_request = /^[.\d]+$/.test(version) ? `${request}@${version}` : request;
-      return callback(
-        null,
-        'module-import ' +
-          (cdn[request as keyof typeof cdn] ?? `https://testingcf.jsdelivr.net/npm/${versioned_request}/+esm`),
-      );
+      return callback();
